read_test.go

```diff
@@ -392,6 +392,135 @@ func TestReadNativeFrames(t *testing.T) {
 	}
 }
 
+func TestReadNativeFrames_BytePixels(t *testing.T) {
```

Comment:
question, in order to test the padded byte case, can we just better parametrize the above test? Esp because the test body seems like it's the same.

One thought could be to introduce a dataBytes parameter into the above test case. If len(data) ==0 in the test case, then we pass dataBytes to binary.Write instead. 

What do you think?

Reply:
I cleaned it up like you suggested, see what you think